MattRowee/OilOnline
# Mr requests

In this PR I have:

-Scaffolded the Request and PaymentType controllers
-Edited Vehicle (Index, Edit, Create) and Request (Index, Edit, Create) views, removing CustomerId and other
unwanted fields ASWELLAS displaying the Name property for Oil Types in select lists and the PlateNumber property for Vehicle select list in Requests.
-added a .Where method to the VehicleController Index method to only display Vehicles registered to the user. 

To Test: 
 - Run git fetch --all to fetch this branch
 - Run git checkout mr-Requests to checkout this branch
- Register a new user and create a vehicle. Your index should display only vehicles you have created on the current user. The forms should not ask for a customerId or mechanicId.
